packages/web/src/components/basic/SelectedFilters.js

```diff
@@ -42,8 +43,78 @@ class SelectedFilters extends Component {
 	};
 
 	clearValues = () => {
-		const { onClear } = this.props;
-		this.props.clearValues();
+		const {
```

Comment:
Similar to my other comment you may wish to deconstruct this huge method into different methods.
This will also make the writing of tests easier and testability/maintainability in general easier.

Reply:
Agreed! 👍
A method named clearValuesToDefault() would be defined and used to clear values to default conditionally.